startup/gui/usd.py

```diff
@@ -45,3 +53,48 @@
 # `texture:format == automatic` isn't well supported at present, so default
 # user-created lights to `latlong`.
 Gaffer.Metadata.registerValue( GafferUSD.USDLight, "parameters.texture:format", "userDefault", "latlong" )
+
+
+def __kindSelectionModifier( targetKind, scene, pathString ) :
+	path = pathString.split( "/" )[1:]
+
+	kind = None
+	while len( path ) > 0 :
+		attributes = scene.attributes( IECore.InternedStringVectorData( path ) )
```

Comment:
Can we omit the `InternedStringVectorData` here, since Murray added a conversion from lists recently?

Reply:
Very nice, fixed in https://github.com/GafferHQ/gaffer/pull/5720/commits/cde068adae72c742305aab275801a2973a37e267